### PR TITLE
Adds PAYG for Amazon Marketplace

### PR DESCRIPTION
**Pull request by fairlydurable**
Signed off from Alex Cort

### CODE REVIEW

File: docs/evaluate/temporal-cloud/pricing.mdx [hunk 1]
```diff
@@ -275,8 +276,12 @@ Basic support has a monthly fee starting from $200.
 
 **Can I purchase Temporal Cloud through my Amazon, Azure, or Google Cloud Platform marketplace?**
 
-You can purchase Temporal Cloud credits in the AWS Marketplace.
-To learn more about our private AWS Marketplace offer, contact our team at [sales@temporal.io](mailto:sales@temporal.io).
+There's two ways to purchase Temporal Cloud through AWS Marketplace:
```
Comment (napcs) [inline docs/evaluate/temporal-cloud/pricing.mdx:279]:
```suggestion
There are two ways to purchase Temporal Cloud through AWS Marketplace:
```

File: docs/evaluate/temporal-cloud/pricing.mdx [hunk 2]
```diff
@@ -275,8 +276,12 @@ Basic support has a monthly fee starting from $200.
...
+
+- Pay-as-you-Go available [here](https://aws.amazon.com/marketplace/pp/prodview-hbcjz7wcl2mvu)
```
Comment (napcs) [inline docs/evaluate/temporal-cloud/pricing.mdx:281]: Check all occurrances of Pay-as-yo-Go for casing and hyphens or spacing and ensure they are consistent. This one differs from the one that follows.

**Review by napcs — DISMISSED** [review]

File: docs/evaluate/temporal-cloud/pricing.mdx (hunk 1 above)
Comment (fairlydurable) [reply docs/evaluate/temporal-cloud/pricing.mdx:279]: Fixing. Thank you for the catch.

File: docs/evaluate/temporal-cloud/pricing.mdx (hunk 2 above)
Comment (fairlydurable) [reply docs/evaluate/temporal-cloud/pricing.mdx:281]: Checking and making them all Pay-As-You-Go, which OpenAI says is correct. Thanks again.

**Comment by fairlydurable** [comment]
docs% grep -i "pay.as" docs/production-deployment/cloud/get-started.mdx
 - Subscribe at the AWS Marketplace for [Temporal Cloud Pay-As-You-Go](https://aws.amazon.com/marketplace/pp/prodview-hbcjz7wcl2mvu).
docs% grep -i "pay.as" docs/evaluate/temporal-cloud/pricing.mdx
To pay for Temporal Cloud with an AWS Account, sign up for  [Temporal Cloud Pay-As-You-Go](https://aws.amazon.com/marketplace/pp/prodview-hbcjz7wcl2mvu) on the AWS Marketplace.
- Pay-As-You-Go available [here](https://aws.amazon.com/marketplace/pp/prodview-hbcjz7wcl2mvu)

**Review by axfelix — APPROVED** [review]

**Pull request merged**